### DISCUSSION TITLE
After removing domains from ALIAS_DOMAINS, domains still in the UI.

### DISCUSSION THREAD

**Discussion by thelittlerocket** [simple-login/app#2100] [Self-hosting question]
Hi there,

some domain was added to the env file as ALIAS_DOMAINS, but now I no longer needed those domains, so I tried to remove them from the ALIAS_DOMAINS and did migration and init command, but after the service started, those domains were still in the UI, is there any way I can remove them completely?

any help would be appreciated.

thanks.

**Comment by nguyenkims** [comment]
The ALIAS_DOMAINS is only used to populate the `public_domain` table. If you want to remove a domain, you need to do the change in this table instead.
